game/src/main/scala/io/aigar/model/Team.scala

```diff
@@ -1,15 +1,86 @@
 package io.aigar.model
 
 import slick.driver.H2Driver.api._
+import slick.lifted.TableQuery
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.Duration
+import scala.concurrent.Await
 
-case class Team(id:Int, teamSecret:String, teamName:String, score:Int)
+case class Team(id: Option[Int], teamSecret: String, teamName: String, score: Int)
 
-class Teams(tag: Tag) extends Table[(Int, String, String, Int)](tag, "TEAMS") {
-  def id = column[Int]("ID", O.PrimaryKey)
+class Teams(tag: Tag) extends Table[Team](tag, "TEAMS") {
+  def id = column[Int]("ID", O.PrimaryKey, O.AutoInc)
   def teamSecret = column[String]("TEAM_SECRET")
   def teamName = column[String]("TEAM_NAME")
   def score = column[Int]("SCORE", O.Default(0))
-  def * = (id, teamSecret, teamName, score)
+  def * = (id.?, teamSecret, teamName, score) <> (Team.tupled, Team.unapply)
+}
+
+object TeamDAO extends TableQuery(new Teams(_)) {
+  lazy val teams = TableQuery[Teams]
+
+  def create(db: Database, team: Team): Team = {
```

Comment:
I changed the name of the functions in this class to make them more self-describing